run_script.sh

```diff
@@ -0,0 +1,123 @@
+
+# Provide a specific name used for identifying the job
+# as it proceeds through Slurm.
+#SBATCH --job-name=example-hpc-alpine-python
+
+# Tells Slurm to gather standard output from running this
+# file and send to a specific file.
+# Special variable symbols may be used here:
+# %j - job ID
+# %a - job array index
+# %A - job array job ID
+#SBATCH --output=example-hpc-alpine-python.out
+
+# Sets a limit on the total time this work may take.
+# The format below is in the form of hours:minutes:seconds.
+#SBATCH --time=01:00:00
+
+# Sets certain Alpine-specific characteristics the Slurm work
+# performed. Can be one of: normal, long, mem.
+# See: https://curc.readthedocs.io/en/latest/running-jobs/job-resources.html
+#SBATCH --qos=normal
+
+# Advises Slurm about the minimum nodes necessary for completing
+# the work included in this script.
+#SBATCH --nodes=1
+
+# Advises Slurm about the maximum number of tasks involved
+# with batch processing.
+#SBATCH --ntasks=4
+
+# Sets an email address to receive notifications from Alpine
+#SBATCH --mail-user=your-email-address-here@cuanschutz.edu
+
+# Indicate which notifications you'd like to receive from Alpine
+# this can also be set to START, END, or FAIL.
+#SBATCH --mail-type=ALL
+
+########################################################
+# Module package commands:
+# ------------------------
+# Next, we use the module package to help load
+# software which is pre-loaded on Alpine.
+########################################################
+
+# unloads all existing modules which may have been previously loaded
+module purge
+
+# use module package to load anaconda software so it may
+# be used by your processes.
+# note: the numbers found after anaconda/####.## are subject
+# to change depending on the versions installed by administrators.
+module load anaconda/2022.10
+
+########################################################
+# Anaconda environment management:
+# ---------------------------------
+# Here we load the Anaconda environment to be used
+# for running the Python code below.
+########################################################
+
+# remove any existing environments that happen to have
+# the same exact name.
+conda env remove --name example_env -y
+
+# next create the environment from the yaml file
+conda env create -f environment.yml
+
+# then activate the environment
+conda activate example_env
+
+########################################################
+# Run a Python file (within Anaconda environment):
+# ------------------------------------------------
+# After loading the environment we run the Python
+# code to perform the work we'd like to accomplish.
+########################################################
+
+# run the python file example.py which takes an argument
+# as a filepath for exporting data which we pass in here
+# from an sbatch exported variable name which is
+# received as a named argparse variable within Python
+# using the same name.
```

Comment:
Maybe clean up or simplify this paragraph. Probably can rely on other comments and the script itself for finer explanation.

Reply:
Thanks, great point! I've updated this to retain some of the information and simplify. My hope here is that one wouldn't mistake the sbatch environment variable for the python argparse value.